device-types/FS/FMU-MC084761EM3.yaml

```diff
@@ -0,0 +1,50 @@
+---
+manufacturer: FS
+model: FMU-MC084761EM3
+slug: fmu-mc084761em3
+comments: '[8ch Dual Fiber 1470-1610nm CWDM Mux Demux + Monitor, Expansion and 1310nm Port END OF SALE](https://www.fs.com/de/products/72428.html)'
```

Comment:
Can you update the line to https://www.fs.com/products/78163.html
So that it goes to the English version. Thanks!

Reply:
sure, sorry about that. i didnt notice that.